### PR TITLE
Add access token attributes integration test

### PR DESCRIPTION
**Pull request by mpmadhavig**
### Purpose
$subject 

### Related Issues
- https://github.com/wso2/product-is/issues/20880

### Related PRs
- https://github.com/wso2/product-is/pull/20905

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/consented/token/OAuth2ServiceAuthCodeGrantJWTAccessTokenWithConsentedTokenColumnTestCase.java [hunk 1]
```diff
@@ -239,13 +239,14 @@ private void validateUserClaims(OIDCTokens oidcTokens) throws JSONException, Par
...
+        // Check if user claims are present in access token.
         JSONObject jwtJsonObject = new JSONObject(new String(Base64.decodeBase64(accessToken.split("\\.")[1])));
-        String email = jwtJsonObject.getString(EMAIL_OIDC_CLAIM);
-        Assert.assertEquals(USER_EMAIL, email, "Requested user claim (Email) is not present in the JWT access "
-                + "token.");
-        Assert.assertTrue(jwtJsonObject.isNull(ADDRESS_OIDC_CLAIM), "Non-consented user claim (address) is"
-                + " present in the JWT access token.");
+        try {
+            Object emailClaim = jwtJsonObject.get(EMAIL_OIDC_CLAIM);
+            Assert.fail("Requested user claim (email) is present in the JWT access token.");
```
Comment (shashimalcse) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/consented/token/OAuth2ServiceAuthCodeGrantJWTAccessTokenWithConsentedTokenColumnTestCase.java:246]: do we need to check if emailClaim is null here? and please check the messages here

Comment (mpmadhavig) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/consented/token/OAuth2ServiceAuthCodeGrantJWTAccessTokenWithConsentedTokenColumnTestCase.java:246]: It's checking whether any user claims are present.

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2AuthorizationCodeGrantJWTTokenTestCase.java [hunk 2]
```diff
@@ -323,10 +324,7 @@ public void testValidateAdditionalUserClaims() {
         applicationConfig.getRequestedClaimList().forEach(claim -> {
             if (authorizingUser.getUserClaims().get(claim) != null) {
                 assertNotNull(accessTokenClaims.getClaim(claim.getOidcClaimUri()),
-                        "Claim " + claim.getOidcClaimUri() + " not found in the access token.");
-                assertEquals(accessTokenClaims.getClaim(claim.getOidcClaimUri()),
-                        authorizingUser.getUserClaims().get(claim),
-                        "Value for claim " + claim.getOidcClaimUri() + " is incorrect in the access token.");
+                        "User claim " + claim.getOidcClaimUri() + " not found in the access token.");
```
Comment (malithie) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2AuthorizationCodeGrantJWTTokenTestCase.java:327]: @mpmadhavig why assert equals is removed ?

Comment (mpmadhavig) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2AuthorizationCodeGrantJWTTokenTestCase.java]: This has been missed during e refactoring effort. Fixed with 52ebe10566c95461912b7fa85477396ca386a1d5

**Review by shashimalcse — DISMISSED** [review]

**Review by Thumimku — APPROVED** [review]
LGTM, let's merge after integration test passed.

**Review by Thumimku — APPROVED** [review]
Let's run the integration tests and merge

**Pull request merged**